ngraph/src/ngraph/validation_util.cpp

```diff
@@ -743,22 +751,14 @@ PartialShape ngraph::infer_slice_shape(const Node* node,
 
                 if (ub < 0)
                 {
-                    ub = std::max(input_shape[input_shape_idx].get_length() + ub, int64_t(0));
+                    ub = std::max(input_shape[input_shape_idx].get_length() + ub,
```

Comment:
Looks like we change logic for an existing operation. @slyalin @evgenylazarev do we need to updated StridedSlice specification? or we need to add new StridedSlice operation to opset?

Reply:
I don't think that we need to add new version of the operation. This is a bug in the implementation which we faced with just one model so far. @mandrono is your change compliant with the specification?

Reply:
The specification says that *Out-of-bounds values will be silently clamped.* So in case negative stride and out of bounds ends we should set upper bound = -1. So, yes, this change compliant with the specification.